server/FakeToOriginalIPConverter/README.md

```diff
@@ -28,7 +32,8 @@ It updates the TcpClient/TcpServer address and port details based on the informa
             "release": "0.0.1",
             "fake-tcp-ip": "1.1.3.2",
             "fake-tcp-port": 3002,
-            "original-tcp-ip": "127.0.0.1",
+            "original-tcp-ip": {
+                "domain-name": "www.domainName.com",
```

Comment:
In Description can we mention to refer second example incase if we need to add domain-name